### PR TITLE
gcp 1.4

### PR DESCRIPTION
**Pull request by PrzemyslawTusinski**
GCP benchmark rule!

```
Policy name                   CIS Google Cloud Platform Foundation    
No. of violations             6                                       
Violations                    
  Resource ID                                                 Rule GUID                                         Rule name                                                   
  ...ount.com/keys/b812601d23e3104be02ef9be99917cee51d4507e"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
  ...ount.com/keys/7485a14020c533797687c4f6b9c8e4304269fcc8"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
  ...ount.com/keys/7d87ee8f4f963e38a95817db4d0e334101faf3fe"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
  ...ount.com/keys/2b01973cb73a9d4d5df5aa7916b236f2bd6b4cb0"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
  ...ount.com/keys/f77242e3e9c06ddbe812cc620ed664c6ae0a5f80"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
  ...ount.com/keys/a0266844e253f2b8991024b7543c95b7abf6f760"  419ab56b-0837-4bc8-a04d-b0e184dc2303              Ensure that there are only GCP-managed service account k... 
```

### CODE REVIEW

File: rules/gcp_service_account_no_user_managed_keys.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+id: 419ab56b-0837-4bc8-a04d-b0e184dc2303
+ruleId: gcp_01
```
Comment (stefansjfw) [inline rules/gcp_service_account_no_user_managed_keys.yaml:2]: let's keep it in sync with aws rules and use 3 digits -> gcp_01 -> gcp_001

**Review by stefansjfw — APPROVED** [review]

File: rules/gcp_service_account_no_user_managed_keys.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+remediation: >
+  From Console
+    1. Go to the IAM page in the GCP Console using https://console.cloud.google.com/iam-admin/iam
+    2. In the left navigation pane, click Service accounts. All service accounts and their corresponding keys are listed.
+    3. Click the service account.
+    4. Click the edit and delete the keys.
+  From CLI To delete a user managed Service Account Key:
+    gcloud iam service-accounts keys delete --iam-account=<user-managed-service-account-EMAIL> <KEY-ID>
+remediationDocURLs:
+  - https://docs.openraven.com/remediations/ensure_AWS_account_security_questions_registered
```
Comment (stefansjfw) [inline rules/gcp_service_account_no_user_managed_keys.yaml:26]: this looks wrong

**Comment by PrzemyslawTusinski** [comment]
@stefansjfw I've added new rule here. Feel free to review it again

File: rules/gcp_service_account_no_user_managed_keys.yaml (hunk 2 above)
Comment (PrzemyslawTusinski) [reply rules/gcp_service_account_no_user_managed_keys.yaml]: Fixed

File: policies/cis_gcp_foundations_benchmark.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+id: d8792397-239e-4028-9539-7b3771ba929f
+policyId: CIS GCP
+policyName:  CIS Google Cloud Platform Foundation
+description: >
+  The CIS Security Benchmarks program provides well-defined, unbiased, consensus-based
+  industry best practices to help organizations assess and improve their security
+enabled: true
+rules:
+  - gcp_service_account_no_user_managed_keys.yaml
+  - gcp_IAM_users__not_assigned_the_Service_Account_User_or_Service_Account_Token_Creator _roles at _level.yaml
```
Comment (stefansjfw) [inline policies/cis_gcp_foundations_benchmark.yaml:10]: file name is bad. should be shorter, and without spaces

Comment (PrzemyslawTusinski) [reply policies/cis_gcp_foundations_benchmark.yaml:10]: What filename you propose here?

Comment (stefansjfw) [reply policies/cis_gcp_foundations_benchmark.yaml:10]: hmm. e.g. `no_project_level_service_account_user_or_token_for_iam_user.yaml'

**Review by stefansjfw — APPROVED** [review]

**Pull request merged**